### ISSUE TITLE
Can't receive frames being sent

### ISSUE THREAD

**Issue by Nimisha-Pabbichetty** [buresu/ndi-python#15]
Hi! I'm sending video frames using send_video.py and trying to receive them using either recv.py or recv_cv.py. On the sending side, it is printed that frames are being sent like this:

![image](https://user-images.githubusercontent.com/60477893/164726206-ba2257d6-0b75-466d-86d9-475a86795018.png)

But on the receiving side, it is only showing 'Looking for sources' when running both files like this:
![image](https://user-images.githubusercontent.com/60477893/164726385-96ea9b08-7687-4d0f-b7ec-db4f5714fa21.png)

What should I do?

**Comment by buresu** [comment]
Hi, Thank you for reporting.
I confirmed same problem on ubuntu 22.04, but on Arch Linux, I didn't have any problems.
This problem seems to be related on avahi, but the solution is not yet known.
I will continue to find the solution.

**Comment by Nimisha-Pabbichetty** [comment]
Thank you for your prompt response. Can you let me know if this issue exists on Windows?

**Comment by buresu** [comment]
Yes, it works fine on windows. The setup steps are no different.
```
# install ndi-python
pip install ndi-python

# run examples
git clone --recursive https://github.com/buresu/ndi-python.git
cd ndi-python/example
pip install -r requirements.txt
python find.py
```

**Comment by buresu** [comment]
Update:
Curiously, it works fine in my current environment.
```
Ubuntu 22.04 LTS
Anaconda 4.12.0
Python 3.10.4
ndi-python 5.1.1.2
avahi-daemon 0.8
```
You can use avahi-discover to make sure that it is working properly.
```
sudo apt install avahi-discover
avahi-discover
```
![Screenshot_20220522_200640](https://user-images.githubusercontent.com/2587873/169692165-fbb35bda-b6e3-4187-b210-c2bfe6d7d729.png)

**Comment by Nimisha-Pabbichetty** [comment]
Interesting, I'll try to match your setup and see if it works.

**Comment by Nimisha-Pabbichetty** [comment]
@buresu I migrated to Windows and it works fine there. I do have a different issue though. I'm transmitting frames of the format 'FOURCC_VIDEO_TYPE_UYVY' but I am only able to receive them in 'RECV_COLOR_FORMAT_BGRX_BGRA' format. The other formats throw an error like this when I try to visualise them:


![image](https://user-images.githubusercontent.com/60477893/176375640-59dbc2e3-0064-4b2a-a34a-f1d479a828ba.png)

**Comment by buresu** [comment]
@Nimisha-Pabbichetty cv.imshow() is not support uyvy format.
If you want to get it in uyvy format, you have to convert it yourself.

You can change the following lines.
```
ndi_recv_create.color_format = ndi.RECV_COLOR_FORMAT_BGRX_BGRA
to
ndi_recv_create.color_format = ndi.RECV_COLOR_FORMAT_FASTEST

and

frame = np.copy(v.data)
to
frame = cv.cvtColor(v.data, cv.COLOR_YUV2BGR_UYVY)
```

**Comment by buresu** [comment]
Closing this issue, but please feel free to reopen if you still have any problems.
